app/assets/javascripts/school_administrator_dashboard/discipline_dashboard/SchoolDisciplineDashboard.js

```diff
@@ -310,7 +310,6 @@ export default class SchoolDisciplineDashboard extends React.Component {
     return (
       <StudentsTable
         rows = {rows}
-        selectedCategory = {this.state.selectedCategory}
         incidentType={"Incidents"}
         resetFn={this.onResetStudentList}/>
```

Comment:
The `resetFn` prop isn't used anymore either, right?

Reply:
Shoot, missed that one.